src/components/AudioTrack/AudioController.vue

```diff
@@ -107,7 +107,7 @@ export default {
     )
     // Sync status from store to parent
     watch(
-      () => [store.state.activeMediaType, store.state.activeMediaId],
+      () => [store.state['active-media'].type, store.state['active-media'].id],
```

Comment:
Does Nuxt automatically use the file name? Can we use the `activeMedia` name for the module to avoid []?

Reply:
Unfortunately, yes, Nuxt uses the file name, and we cannot use the camelCased version. We could rename the store module to `active`

Reply:
That'll be a good idea! I would've liked if Nuxt offered to option to name the state but we've got to work with what we have. 

`state.active` will be better than `state['active-media']`.